protocol/messenger.go

```diff
@@ -2917,7 +2917,7 @@ func (m *Messenger) RetrieveAll() (*MessengerResponse, error) {
 		return nil, err
 	}
 
-	return m.handleRetrievedMessages(chatWithMessages)
+	return m.handleRetrievedMessages(chatWithMessages, true)
```

Comment:
Wouldnt this cause everyone to store the waku messages automatically if they're are admins of a community?
Will this be an option in the future, or should admins be required always to store the messages? 
i.e. I might want to store messages if i'm using my desktop machine, but on mobile i might not.

Reply:
Right so I had a talk with @John-44 about this and we decided that admins should always store every community message as waku message. However we didn't talk about how this should behave on mobile (where the torrent protocol is turned off by default). So this is a valid concern..

One alternative would be to always put storing the waku messages behind the same flag of whether the protocol is enabled or not. That would apply the same on mobile and desktop.

Just means that, if admins turn off the protocol for whatever reason, they will not store messages as waku messages, resulting in them not being part of the archives.